BraveRewardsUI/Create Wallet/CreateWalletView.swift

```diff
@@ -105,17 +113,36 @@ extension CreateWalletViewController {
         $0.leading.trailing.equalTo(self).inset(50.0)
         $0.height.equalTo(UX.createWalletButtonHeight)
       }
+      termsOfServiceLabel.snp.makeConstraints {
+        $0.top.equalTo(self.createWalletButton.snp.bottom).offset(25.0)
+        $0.leading.trailing.equalTo(self).inset(60.0)
+      }
       learnMoreButton.snp.makeConstraints {
-        $0.top.equalTo(self.createWalletButton.snp.bottom).offset(30.0)
+        $0.top.equalTo(self.termsOfServiceLabel.snp.bottom).offset(30.0)
         $0.centerX.equalTo(self)
         $0.height.equalTo(UX.learnMoreButtonHeight)
         $0.bottom.equalTo(self).offset(-20.0)
       }
+      
+      termsOfServiceLabel.onLinkedTapped = onLinkTapped
     }
     
     @available(*, unavailable)
     required init(coder: NSCoder) {
       fatalError()
     }
+    
+    private func onLinkTapped(_ link: URL) {
```

Comment:
This should actually be handled at a controller level (i.e. `CreateWalletController`), and should call the Rewards delegate method that asks the client to open a URL (`RewardsUIDelegate` → `loadNewTabWithURL(_:)`).

Also, the URLs are: 
- ToS: https://brave.com/terms-of-use/
- PP: https://brave.com/privacy/#rewards

Reply:
Fixed.